AIPscan/Reporter/views.py

```diff
@@ -133,16 +136,30 @@ def view_file(file_id):
 
 @reporter.route("/reports/", methods=["GET"])
 def reports():
-    """Reports returns a standard page in AIPscan that lists the
-    in-built reports available to the caller.
+    """Reports page lists available built-in reports
+
+    Storage Service ID can be specified as an optional parameter. If
+    one is not specified, use the default Storage Service.
     """
+    storage_service_id = request.args.get("amss_id")
+    if storage_service_id is not None:
+        storage_service = StorageService.query.get(storage_service_id)
+    else:
+        storage_service = StorageService.query.filter_by(default=True).first()
     all_storage_services = StorageService.query.all()
+    file_formats = storage_service.unique_file_formats
+    # Sort PUIDs "naturally" - i.e. ["fmt/1", "fmt/2", "fmt/10"] rather
+    # than Python's default ["fmt/1", "fmt/10", "fmt/2"].
+    puids = natsorted(storage_service.unique_puids)
```

Comment:
I see your point about the self-describing name. I added the comment because "natural" is a rather vague term, so I wasn't sure if people would know what it meant without the clarification even if they could glean that `natsorted` means "naturally sorted". I'm fine with separating it into its own helper function if that helps clarify intent. I see what you mean about extensibility as well, though I wonder if that's a bit premature.

On adding this directly into the `render_template` args, I see your point but I think this might be complicated by https://github.com/artefactual-labs/AIPscan/pull/76#discussion_r511143890, namely needing some sort of exception handling for `AttributeError`s when we don't have a Storage Service.

Reply:
> I see what you mean about extensibility as well, though I wonder if that's a bit premature.

I am really just focusing on what is clean and self-describing here but in terms of constructive code-review I just want to make sure I share what I do perceive to be benefits. I'm not at all suggesting we will be extending any of this right now.

Reply:
> in terms of constructive code-review I just want to make sure I share what I do perceive to be the benefits if we go the extra step ... I'm just trying to use the comment as a cue to say, okay, this is useful to one of us, how do we improve on that?

Understood, and I appreciate the constructive criticism! I appreciate the nudge to improve on the execution.